### ISSUE TITLE
Todo: New mob textures

### ISSUE THREAD

**Issue by Sunconure11** [Sunconure11/dungeon-mobs#26]
A reminder to myself.

**Comment by Sunconure11** [comment]
Will do this in september

**Comment by BlackJar72** [comment]
I'm looking forward to see what you do.

What time I put into modding in the near future will probably go toward re-writing climatic biomes rather then enhancing dungeon mobs.

**Comment by Sunconure11** [comment]
Starting on one texture now, jesus, these models are complicated.

**Comment by BlackJar72** [comment]
Yeah, they are -- and to make things worse, they all use the same texture size a a player -- not really right for most of models.

**Comment by Sunconure11** [comment]
Hm, I wonder if you could tweak the models so that texturing them is easier. Such as expanding their texture size and setting all of the pieces of the texture map in new spots.

**Comment by Sunconure11** [comment]
Unless a remodel of the current models can be done, due to how awkwardly they set up, I am unable to do this at this current point in time.

**Comment by BlackJar72** [comment]
I think new models would have to be made from scratch, which is something I suck at (note: the creatures I added were two basic humanoids using the vanilla model, a semi-humanoid with few pieces, and creature with a mostly procedural model).

I did try to make some changes to the beholder so its hit box wouldn't extend under the mob -- but all I could do is mess things up.  I do think that to do this someone would probably have to learn one of the modelling programs / mods and just make new models from scratch.  I'm not sure I could really be much help with this even if I did have reasons to step away from this project right now.

I'm guess GnomeWorks used Techne or something like it to make the originals.

(BTW, if you are near the Hurricane area you probably won't see this, but be safe.)
